### ISSUE TITLE
Missing sound when play videos from history tab on youtube

### ISSUE THREAD

**Issue by Dragon31337** [AndreyPavlenko/Fermata#441]
In most cases when I play a video from the history tab on YouTube video starts without sounds (maybe as a part of anti-ads measures). The usual fix is to go back and launch it from there one more time, in most cases that fixes the issue.
It does not happen is much when I run videos from main youtube page.

Version: 1.9.9 and previous

**Comment by AndreyPavlenko** [comment]
Have you tried to disable the skip ads option in the youtube addon settings?

**Comment by Dragon31337** [comment]
No, I did not that would be such torture. I wish there were separate settings "mute ads" and "skip ads"...

**Comment by Dragon31337** [comment]
The repro rate with videos player from the history tab is about 90% if Fermata was not run for some time, like a day. While skipping adds there is a popup in the right-top corner to unmute sound, if I manage to click it sound is always there. Is there an option to separate settings into "skip ads" and "mute ads" at least?
